src/Test.UnitTests.Sarif/TestData/JsonConverters/LocationId_BigInteger.sarif

```diff
@@ -0,0 +1,148 @@
+{
+  "runs": [
+    {
+      "tool": {
+        "driver": {
+          "name": "mobsfscan",
+          "rules": [
+            {
+              "id": "android_detect_tapjacking",
+              "name": "AndroidDetectTapjacking",
+              "helpUri": "https://github.com/MobSF/owasp-mstg/blob/master/Document/0x05h-Testing-Platform-Interaction.md#testing-for-overlay-attacks-mstg-platform-9"
+            }
+          ],
+          "version": "0.0.8",
+          "informationUri": "https://github.com/MobSF/mobsfscan",
+          "semanticVersion": "0.0.8"
+        }
+      },
+      "invocations": [
+        {
+          "executionSuccessful": true,
+          "endTimeUtc": "2022-01-16T17:04:25Z"
+        }
+      ],
+      "results": [
+        {
+          "message": {
+            "text": "This app does not have capabilities to prevent tapjacking attacks. An attacker can hijack the user's taps and tricks him into performing some critical operations that he did not intend to."
+          },
+          "level": "note",
+          "locations": [
+            {
+              "id": 31197130097450771296369962162453149327732752356239421572342053257324632475324,
```

Comment:
>31197130097450771296369962162453149327732752356239421572342053257324632475324 [](http://example.com/codeflow?start=20&length=77)

data in this file contains very long int, 64 int max +1, 32 int max +1 and a few normal ones.